code/extensions/ControllerSubsites.php

```diff
@@ -15,7 +15,7 @@ public function controllerAugmentInit()
     {
         if ($subsite = Subsite::currentSubsite()) {
             if ($theme = $subsite->Theme) {
-                SSViewer::set_themes([$theme, SSViewer::DEFAULT_THEME]);
+                SSViewer::add_themes([$theme]);
```

Comment:
You don't want $theme to be after the default; Otherwise your theme may not even get activated for many templates!